tern/__main__.py

```diff
@@ -167,15 +167,13 @@ def main():
     parser_report.add_argument('-d', '--dockerfile', type=check_file_existence,
                                help="Dockerfile used to build the Docker"
                                " image")
-    parser_report.add_argument('-i', '--docker-image',
-                               help="Docker image that exists locally -"
-                               " image:tag"
-                               " The option can be used to pull docker"
-                               " images by digest as well -"
-                               " <repo>@<digest-type>:<digest>")
+    parser_report.add_argument('-i', '--image',
+                               help="A container image referred either by "
+                               " image:tag or image@digest-type:digest")
```

Comment:
Is the default registry to pull from dockerhub? If so, do we want to mention that?

Reply:
Yes. Skopeo uses Docker's resolver. We didn't mention that before...

Reply:
Oh I see what you mean. I'll use repo instead of image.

Reply:
I think image is more clear to users...